src/Test/Utilities/Portable/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -274,19 +273,19 @@ private void AnalyzeOperation(OperationAnalysisContext operationContext)
                     ((ILocalReferenceOperation)advanceAssignment.Target).Local == testVariable)
                 {
                     // Advance binary operation is known to involve a reference to the local used in the test and a constant.
-                    advanceIncrement = CreateIncrementOneLiteralExpression(semanticModel, advanceAssignment);
+                    advanceIncrement = CreateIncrementOneLiteralExpression(advanceAssignment);
                     advanceOperationCode = BinaryOperatorKind.Add;
                 }
             }
         }
 
-        private static ILiteralOperation CreateIncrementOneLiteralExpression(SemanticModel semanticModel, IIncrementOrDecrementOperation increment)
+        private static ILiteralOperation CreateIncrementOneLiteralExpression(IIncrementOrDecrementOperation increment)
         {
             string text = increment.Syntax.ToString();
             SyntaxNode syntax = increment.Syntax;
             ITypeSymbol type = increment.Type;
             Optional<object> constantValue = new Optional<object>(1);
-            return new LiteralExpression(semanticModel, syntax, type, constantValue, increment.IsImplicit);
+            return new LiteralExpression(((Operation)increment).OwningSemanticModel, syntax, type, constantValue, increment.IsImplicit);
```

Comment:
>return new LiteralExpression(((Operation)increment).OwningSemanticModel, syntax, type, constantValue, increment.IsImplicit); [](start = 12, length = 124)

This is a preexisting condition, but I am not sure why do we need to create IOperation nodes here. It looks like the consumer is only interested in a constant value. Why don't we return that instead?